addons/sale/views/sale_order_views.xml

```diff
@@ -261,13 +261,15 @@
                     type="object"
                     data-hotkey="shift+g"
                     class="oe_highlight"
+                    groups='account.group_account_invoice'
```

Comment:
As said in the other comment, could be the `sales_team.group_sale_salesman` group instead.
Also, please use double quoted strings in xml